search.go

```diff
@@ -2,6 +2,7 @@ package registry
 
 import (
 	"fmt"
+	"strings"
 )
 
 // Searchable is an interface for supporting search
```

Comment:
This isn't really part of your changes, but while we're here this comment might be better phrased as:
"Searchable is an opt-in interface for registries that wish to support search"